meson.build

```diff
@@ -1227,7 +1227,7 @@ endif
 
 # vulkan
 vulkan_opt = get_option('vulkan').require(
-    libplacebo.found(),
+    libplacebo.get_variable('pl_has_vulkan') == '1',
```

Comment:
I think this one might need to be `libplacebo.found() and libplacebo.get_variable...` in the event libplacebo isn't found.

Reply:
Yes, you are right. Or we need to add a `default_value: '0'` to it.